virtwho/parser.py

```diff
@@ -112,21 +120,151 @@ def check_argument_consistency(cli_options):
     Final check of cli options that can not be done in custom actions.
     """
 
+    # These options can be required
+    REQUIRED_OPTIONS = ['owner', 'env', 'server', 'username']
```

Comment:
This part of refactoring was done, because current implementation uses one function for checking consistency of environment variables ad command line arguments. Thus error logs can be confusing.